editor/editor_interface.h

```diff
@@ -54,12 +54,25 @@ class Texture2D;
 class Theme;
 class VBoxContainer;
 class Window;
+class SceneTreeDialog;
+class PropertySelector;
 
 class EditorInterface : public Object {
 	GDCLASS(EditorInterface, Object);
 
 	static EditorInterface *singleton;
 
+	// Editor dialogs.
+
+	SceneTreeDialog *dialog_scene_tree = nullptr;
+	PropertySelector *dialog_property_selector = nullptr;
+
+	void _node_selected(const NodePath &p_node_paths, const Callable &p_callback);
+	void _node_selection_canceled(const Callable &p_callback);
+	void _property_selected(const String &p_property_name, const Callable &p_callback);
+	void _property_selection_canceled(const Callable &p_callback);
+	static void call_popup_dialog_callback(const Callable &p_callback, const Variant &p_selected, const String &p_context);
```

Comment:
```suggestion
	static void _call_popup_dialog_callback(const Callable &p_callback, const Variant &p_selected, const String &p_context);
```
Why is this static?

Reply:
It's a helper method that doesn't depend on the state of the `EditorInterface`. Would there be an advantage to making it making it non-static?

Reply:
We are usually only making methods static if they are intended to be called from the outside and without requiring a class instance (`get_singleton()` is the most common one). This method is private, so it makes little sense for it to be static.